docs/sources/explore/_index.md

```diff
@@ -57,6 +60,10 @@ cards:
 
 Explore is your starting point for querying, analyzing, and aggregating data in Grafana. You can quickly begin creating queries to start analyzing data without having to create a dashboard or customize a visualization.
 
+{{% admonition type="note" %}}
+If you're using Mimir, Loki, Tempo or Pyroscope, be sure to try our [simplified queryless Explore experiences]({{< relref "simplified-exploration" >}}).
```

Comment:
In the technical docs, we try to [minimize "we" and "our"](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#guidelines-for-whats-new-content): "Our style normally prefers second person (“you”) and avoids first person (“I”/“we”)."
```suggestion
If you're using Mimir, Loki, Tempo, or Pyroscope, try the [simplified queryless Explore experiences]({{< relref "simplified-exploration" >}}).
```